packages/frontend/src/components/common/Filters/FilterInputs/AutoComplete/autoCompleteUtils.ts

```diff
@@ -31,10 +31,7 @@ export const mergeUniqueValues = (
     values: string[],
     newValues: string[],
 ): string[] => {
-    const normalizedValues = [
```

Comment:
@IrakliJani do you remember why we were lowercasing all the values? Since our filters are case-sensitive it makes sense for us to not lower case the values.

Reply:
@ZeRego I am not sure, now that I think of I think it's totally unnecessary.
@muhsinkamil can you remove all code that is responsible of lowercasing?

`const normalize = (item: string) => item.toLocaleLowerCase();` you can remove this and follow the ts compiler. thanks!